networkx/drawing/tests/test_layout.py

```diff
@@ -380,6 +388,15 @@ def test_spiral_layout_equidistant(self):
         dist = np.linalg.norm(p[1:] - p[:-1], axis=1)
         assert np.allclose(np.diff(dist), 0, atol=1e-3)
 
+    def test_forceatlas2_layout(self):
+        # check whether imporatial pos input still returns a proper position
+        G = self.Gs
+        pos = nx.random_layout(G)
+        del pos[list(G.nodes())[0]]
+        pos = nx.forceatlas2_layout(G, pos = pos)
```

Comment:
```suggestion
        # check whether partial pos input still returns a proper position
        G = self.Gs
        node = nx.utils.arbitrary_element(G)
        pos = nx.circular_layout(G)
        del pos[node]
        pos = nx.forceatlas2_layout(G, pos=pos)
```
Changes here:  
- spell check comment
- remove random nature of the test (so it doesn't become intermittently failing).
- use `nx.utils.arbitrary_element`
- style requirement `pos=pos` no space around `=` for keyword args.

Reply:
Thanks! Will have to check my linter again ;-)!